pyrene/package.json

```diff
@@ -32,74 +32,73 @@
     "updateIconfont": "bash -c 'cat $0 | fgrep -A3 .icon- | sed -E '\"'\"' s/\\.icon-(.*):before /:global(.pyreneIcon-\\1:before)/ '\"'\"' >  src/styles/iconfont.css; cp $1 src/fonts/pyrene-iconfont.woff;'"
   },
   "devDependencies": {
-    "@babel/core": "^7.11.6",
-    "@babel/plugin-proposal-class-properties": "^7.10.4",
-    "@babel/plugin-proposal-object-rest-spread": "^7.11.0",
-    "@babel/plugin-transform-object-assign": "^7.10.4",
-    "@babel/preset-env": "^7.11.5",
-    "@babel/preset-react": "^7.10.4",
-    "@babel/preset-typescript": "^7.12.7",
-    "@babel/register": "^7.11.5",
-    "@osag/eslint-config": "^1.4.0",
+    "@babel/core": "^7.13.14",
```

Comment:
Ah! I see! This PR actually already updates most of the lagged behind dependencies :) that's so great 👍  Thank you so much!

For libs such as react-select which we need more testing before upgrading, it would be nice to have a list of them and can tackle each one separately :)